Add ResNet50 PyTorch models at various batch sizes

mariecwhite opened the pull request:
Bert-Large is part of our progression suite that we will be monitoring closely at various batch sizes.

Adds a negligible amount of running time to the CUDA benchmarks. x86 CPU unaffected.

pzread commented on build_tools/python/e2e_test_framework/models/torch_models.py:
```diff
@@ -52,3 +52,94 @@
+    entry_function="forward",
+    input_types=["8x3x224x224xf32"])
+
+RESNET50_16x3x224x224_FP32_TORCH = common_definitions.Model(
```
nit: "x" in the middle should be capitalized?

mariecwhite replied:
Changed to X

mariecwhite committed:
Add ResNet50 PyTorch models at various batch sizes

ResNet50 is part of our progression suite that we will be monitoring closely at various batch sizes.

Adds a negligible amount of running time to the CUDA benchmarks. x86 CPU unaffected.
